evaluate/report/markdown.go

```diff
@@ -0,0 +1,106 @@
+package report
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"text/template"
+	"time"
+
+	pkgerrors "github.com/pkg/errors"
+	"github.com/zimmski/osutil/bytesutil"
+
+	"github.com/symflower/eval-dev-quality/evaluate/metrics"
+)
+
+// Markdown holds the values for exporting a Markdown report.
+type Markdown struct {
+	// DateTime holds the timestamp of the evaluation.
+	DateTime time.Time
+	// Version holds the version of the evaluation tool.
+	Version string
+
+	// CSVPath holds the path of detailed CSV results.
+	CSVPath string
+	// LogPath holds the path of detailed logs.
+	LogPath string
+	// SVGPath holds the path of the charted results.
+	// REMARK The charts will be generated during the export.
+	SVGPath string
+
+	// CategoriesPerModel holds
+	CategoriesPerModel map[string]metrics.Assessments
+	// TotalScore holds the total score.
```

Comment:
total score of what?

Reply:
Yeah since we don't know yet I thought leave it vague xD but I will try to be more precise.